log-skeleton/src/components/content.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styles from '../styles/Content.module.css'
+
+const Content = () => {
+
+    return (
+        <div className={styles.content}>
+            <div className={styles.leftPanel}>
+                <h4>Left</h4>
+            </div>
+
+            <div className={styles.midPanel}>
+                <h4>Mid</h4>
+            </div>
+
+            <div className={styles.leftPanel}>
```

Comment:
Should the style be rightPanel? Otherwise if left and right use the same style maybe rename it to sidePanel or similar

Reply:
Fixed it 👍🏽